Nos/Views/Discover/DiscoverTab.swift

```diff
@@ -37,6 +37,7 @@ struct DiscoverTab: View {
             )
             .autocorrectionDisabled()
             .onSubmit(of: .search) {
+                analytics.searchedDiscover()
```

Comment:
I'm not sure how valuable this is given Matt's comment: https://github.com/planetary-social/nos/issues/1304#issuecomment-2229191372

It seems like this could actually be misleading; we might think only a small number of users are searching in the app when in reality every time a user enters three or more characters, we're searching relays.

Reply:
The acceptance criteria is misleading then, it just mentions that problem you say for the second analytics event, not this one. The first event explicitly mentions that we should track the enter key. I’ll be happy to change this, but I’m unsure what @setch-l actually wants to track. I hope I’m clear, I just want to be sure what the AC is.

Reply:
@martindsq You're right; the acceptance criteria are misleading. I think what happened is that Matt discovered that the criteria might not be useful as written, but I haven't seen a response from Linda. I'd just ask her.

Reply:
I updated the code to follow the suggestion @mplorentz made in the ticket.